src/components/NavigationBar/index.tsx

```diff
@@ -1,7 +1,10 @@
 import { Navbar, Nav } from 'react-bootstrap';
 import ShoppingCart from './ShoppingCart';
 
-const NavigationBar = () => {
+interface NavProps {
+  length: number;
+}
+const NavigationBar = ({ length }: NavProps) => {
```

Comment:
`length` is weird. Use `cartItemsCount` or something informative

Reply:
updated